cheri_compressed_cap_common.h

```diff
@@ -642,29 +752,37 @@ static bool _cc_N(fast_is_representable_new_addr)(bool sealed, _cc_addr_t base,
     }
 #undef MOD_MASK
 
-    return ((inRange && inLimits) || (e >= highest_exp));
+    return ((inRange && inLimits) || (e >= highest_exp - 2));
```

Comment:
Is this change valid for non-Morello?

Reply:
This is what it should have always been for both (I think)